esmvalcore/cmor/_fixes/shared.py

```diff
@@ -0,0 +1,25 @@
+"""Shared functions for fixes."""
+import iris
+import numpy as np
+from cf_units import Unit
+
+
+def add_height_coord(cube):
+    """Add height coordinate to cube."""
+    height_coord = iris.coords.AuxCoord(2.0,
```

Comment:
Maybe you can specify the 2m as a function argument? Because there are also other height coordinates (e.g. 10m, 100m) defined in the cmor table, so it could be useful to make this function slightly more generic.

Reply:
Done